docs/source/resources/contributing/cursor/cursor-rules-reference.md

```
@@ -27,11 +27,13 @@ This document provides a comprehensive reference for all available Cursor rules
 **Purpose**: Overarching standards for all source, test, documentation, and CI files.
 
 **Prompt**:
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)